### PR TITLE
Looking into ZAS gas leaks

### PR DESCRIPTION
**Pull request by MadmanMartian**

### CODE REVIEW

File: code/ZAS/Turf.dm [hunk 1]
```diff
@@ -67,23 +67,20 @@
...
 
-/turf/simulated/update_air_properties()
+/turf/simulated/update_air_properties(var/verbose = TRUE)
 	if(zone && zone.invalid)
 		c_copy_air()
 		zone = null //Easier than iterating through the list at the zone.
 
 	var/s_block = c_airblock(src)
-	if(s_block & AIR_BLOCKED)
+	if(s_block & AIR_BLOCKED && !locate(/obj/machinery/door/airlock) in src)
```
Comment (MadmanMartian) [inline code/ZAS/Turf.dm:76]: even more hacky than before. Will continue

Comment (Probe1) [reply code/ZAS/Turf.dm:76]: I like your style.

**Comment by DamianX** [comment]
when

File: code/ZAS/Turf.dm [hunk 2]
```diff
@@ -67,23 +67,20 @@
...
 		#ifdef ZASDBG
-		to_chat(if(verbose) world, "Self-blocked.")
```
Comment (ghost) [inline code/ZAS/Turf.dm:78]: what the fuck why did this work to begin with

Comment (MadmanMartian) [reply code/ZAS/Turf.dm:78]: #ifdef doesnt even look at the code inbetween if the thing isnt defined

Comment (ghost) [reply code/ZAS/Turf.dm:78]: god damn

**Comment by DamianX** [comment]
dead

**Pull request closed without merging**